include/linker/common-ram.ld

```diff
@@ -170,14 +170,6 @@
 		__net_if_dev_end = .;
```

Comment:
Seems fine to me. 

However, to de-clutter net_core and net_if and avoid scattering TC stuff: maybe centralizing most of TC functions into net_tc.c would be nice